packages/promo/components/PromoBanner.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import { cx } from "emotion";
+import { designTokens as dt, SpacingBox } from "../../";
+import PromoContent from "./PromoContent";
+import { PromoProps, GradientStyle } from "../types";
+import { getBackgroundGradient, gradientStyles } from "../style";
+
+interface PromoBannerProps extends PromoProps {
+  // TODO: confirm with designers that the gradient background is something we will reuse
+  // If so, make a GradientPromoBanner component and write guidelines on how/when it gets used.
+  // Otherwise, we shouldn't ever be doing this.
+  /** Whether a blue gradient should be rendered in the banner's background. */
```

Comment:
Why blue?

Reply:
A blue-to-grey gradient is just what the design team specified. These comments are out of date because I copied most of this over from Kommander.